docs/resources/tozny_realm_application_access_control.md

```diff
@@ -0,0 +1,135 @@
+}
+# A resource for creating an OIDC based realm application
+resource "tozny_realm_application" "jenkins_oidc_application" {
+  # Block on and use client credentials generated from the provisioned account
+  depends_on = [
+    tozny_realm.my_organizations_realm,
+  ]
+  client_credentials_config = tozny_account.autogenerated_tozny_account.config
+  realm_name                = tozny_realm.my_organizations_realm.realm_name
+  client_id                 = "jenkins-oid-app"
+  name                      = "Jenkins"
+  active                    = true
+  protocol                  = "openid-connect"
+}
+
+# A resource for adding Access Control policy for a Application
+resource "tozny_realm_application_access_control" "jenkins_application_access_control" {
+  # Block on and use client credentials generated from the provisioned account, application creation and group creation
+  depends_on = [
+    tozny_realm.my_organizations_realm,
+    tozny_realm_application.jenkins_oidc_application,
+    tozny_realm_group.my_first_group,
+    tozny_realm_group.my_second_group
+  ]
+  client_credentials_config = tozny_account.autogenerated_tozny_account.config
+  realm_name                = tozny_realm.my_organizations_realm.realm_name
+  application_id            = tozny_realm_application.jenkins_oidc_application.application_id
+  enabled                   = true
+  group {
+    group_id           = tozny_realm_group.my_first_group.group_id
+    extend_to_children = false
+
+  }
+  group {
+    group_id           = tozny_realm_group.my_second_group.group_id
+    extend_to_children = true
+
+  }
+}
+
+
+```
+
+## Argument Reference
+
+### Top-Level Arguments
+
+- `client_credentials_filepath` - (Optional) The filepath to Tozny client credentials for the provider to use when provisioning this resource. For this resource either this value or both `account_username` and `account_password` must be set on the provider. Omit if using `client_credentials_config`.
+- `client_credentials_config` - (Optional) A JSON string containing Tozny client credentials for the provider to use when provisioning this resource. For this resource either this value or both `account_username` and `account_password` must be set on the provider. Omit if using `client_credentials_filepath`.
+- `realm_name` - (Required) The name of the Realm to provision the Application for.
+- `application_id` - (Required) Server defined unique identifier for the Application.
+- `enabled` - (Required) Whether this application has managed access control.
+- `group` - (Optional) Users within the selected groups can access this application.
+
+### Group Schema
+
+- `group_id` - (Required) Service defined unique identifier for the group.
+- `extend_to_children` - (Optional) Wether SubGroups are able to access this application as well. Defaults to `false`.
+
+## Attribute Reference
+
+- `id` - Server defined unique identifier for the Application.
```

Comment:
nit: prefer generating a random id vs tying this unique identifier to the application that the access control is for

Reply:
Easy fix!